test/Microsoft.AspNet.Mvc.FunctionalTests/TestHelper.cs

```diff
@@ -114,7 +117,10 @@ public static TestServer CreateServer(Action<IApplicationBuilder> builder, strin
                 applicationBasePath,
                 applicationWebSiteName);
             services.AddInstance<IApplicationEnvironment>(environment);
-            services.AddInstance<IHostingEnvironment>(new HostingEnvironment(environment));
+            var hostingEnvironment = new HostingEnvironment();
```

Comment:
Maybe we should consider a helper to initialize all 3 of these like the ctor used to with appEnv, hostingEnv.Initialize(applicaitonBasePath) that will set WebRoot/FileProvider?